packages/connect-conformance/package.json

```diff
@@ -1,43 +1,49 @@
 {
   "name": "@connectrpc/connect-conformance",
   "private": true,
-  "bin": {
-    "conformancenodeserver": "bin/conformancenodeserver",
-    "conformancenodeclient": "bin/conformancenodeclient",
-    "conformancewebclient": "bin/conformancewebclient",
-    "conformancecloudflareserver": "bin/conformancecloudflareserver",
-    "connectconformance": "bin/connectconformance"
+  "main": "./dist/cjs/index.js",
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.js"
+    },
+    "./node": {
+      "import": "./dist/esm/node/index.js",
+      "require": "./dist/cjs/node/index.js"
+    },
+    "./web": {
+      "import": "./dist/esm/web/index.js",
+      "require": "./dist/cjs/web/index.js"
+    }
```

Comment:
For a solid separation of concerns, this package should provide the basics for testing, but it should include any of the code under test. Code under test belongs into a `conformance` directory of a package we want to test.